packages/hooks/src/useTotalAccountBalance.ts

```diff
@@ -14,19 +14,17 @@ type TUseTotalAccountBalance = {
 
 const useTotalAccountBalance = (accounts: TUseTotalAccountBalance[]) => {
     const total_assets_real_currency = useRealTotalAssetCurrency();
-    const { handleSubscription, exchange_rates } = useExchangeRate();
+    const { handleSubscription, getExchangeRate } = useExchangeRate();
 
     if (!accounts.length) return { balance: 0, currency: total_assets_real_currency };
 
     const balance = accounts.reduce((total, account) => {
         const new_base = account?.account_type === 'demo' ? 'USD' : total_assets_real_currency || '';
         const new_target = account.currency || total_assets_real_currency || '';
 
-        let new_rate = 1;
-        if (new_base === '' || new_target === '') new_rate = 1;
-        else if (new_base !== new_target) handleSubscription(new_base, new_target);
+        if (new_base !== new_target) handleSubscription(new_base, new_target);
 
-        if (exchange_rates && exchange_rates[new_base]) new_rate = exchange_rates[new_base][new_target] || 1;
+        const new_rate = getExchangeRate(new_base, new_target);
```

Comment:
@sergei-deriv  why did you remove this fallback exchange _rate = 1 ?

Reply:
@mahdiyeh-deriv it's because `getExchangeRate` returns 1 if it doesn't find rate in the `exchangeRates` object